src/box/lua/call.c

```diff
@@ -1268,6 +1269,56 @@ lbox_func_new_or_delete(struct trigger *trigger, void *event)
 	return 0;
 }
 
+static int
+lbox_grant_rt_access(struct lua_State *L)
+{
+	if (lua_gettop(L) != 2)
+		return luaL_error(L, "internal.grant_lua_call bad arguments. "
+				  "Expected exactly two: user name and "
+				  "function name.");
+
+	struct credentials *grantor = effective_user();
+	struct user *grantor_user = user_find(grantor->uid);
+	if (grantor->uid != ADMIN && is_super(grantor_user) == 0)
+		return luaL_error(L, "internal.grant_lua_call only ADMIN or "
+				  "user with role SUPER can grant access.");
+
+	size_t grantee_name_len;
+	const char *grantee_name = luaL_checklstring(L, 1, &grantee_name_len);
+
+	size_t func_name_len;
+	const char *func_name = luaL_checklstring(L, 2, &func_name_len);
+
+	grant_rt_access(grantee_name,  (uint32_t)grantee_name_len,
+			func_name, (uint32_t)func_name_len);
+	return 0;
+}
+
+static int
+lbox_revoke_rt_access(struct lua_State *L)
```

Comment:
AFAIU, it is not used in the final version of the #10392 patch. It means that is it a dead code. I suggest to drop it.

At the same time #10392 adds `rt_lua_call_access_reset()`. It seems logical to add it in this patch instead.

Reply:
`rt_lua_call_access_reset` added, `lbox_revoke_rt_access` removed.